### PR TITLE
onnx v1.15.0

### PR DESCRIPTION
**Pull request by adityagoel4512**
Follow up to https://github.com/conda-forge/onnx-feedstock/pull/102.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by adityagoel4512** [comment]
@conda-forge-admin, please rerender

File: recipe/0001-Link-to-abseil_dll.patch [hunk 1]
```diff
@@ -1,17 +1,8 @@
-From b2828440f9930c76e42235ddccfa0954d58a84ac Mon Sep 17 00:00:00 2001
-From: "Uwe L. Korn" <uwe.korn@quantco.com>
-Date: Tue, 4 Jul 2023 11:29:21 +0200
-Subject: [PATCH] Link to abseil_dll
-
----
- CMakeLists.txt | 2 +-
```
Comment (hmaarrfk) [inline recipe/0001-Link-to-abseil_dll.patch:7]: i think you should attempt to keep the attribution of this patch.

Comment (cbourjau) [reply recipe/0001-Link-to-abseil_dll.patch:7]: @xhochy Is there some official guideline to include such a header in the patch?

Comment (adityagoel4512) [reply recipe/0001-Link-to-abseil_dll.patch:7]: What is the reason for this @hmaarrfk? The original author can be determined through git ultimately.

**Review by cbourjau — APPROVED** [review]

**Pull request merged**